### PR TITLE
Iss-150: Fixed section about discarding local uncommitted changes

### PR DESCRIPTION
**Pull request by hgautham**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -295,25 +295,38 @@ $ git stash pop
...
 
-To reset only a specific file, you can use that filename as the argument:
+This will revert all local uncommitted changes (should be executed in repo root):
 
 ```sh
-$ git reset filename
+$ git checkout .
+```
+
+You can also revert uncommitted changes only to particular file or directory:
```
Comment (RichardLitt) [inline README.md]: to a particular file

**Comment by RichardLitt** [comment]
Looks pretty good. One small grammar edit suggested.

**Comment by hgautham** [comment]
Updated the PR with the minor grammar edit @RichardLitt

**Comment by RichardLitt** [comment]
Thank you, @hgautham. @flexdinesh note - I'm merging this. You may need to rebase, but I think some of your edits will still be useful.

**Pull request merged**